src/main/java/CancelGPT/command/Command.java

```diff
@@ -0,0 +1,23 @@
+package CancelGPT.command;
+
+public enum Command {
```

Comment:
Good idea using Enums for Commands as there is a fixed set of Commands.
Upper case for all the names is a very good idea as it matches the Naming Conventions where constants should be all in Uppercase.